### DISCUSSION TITLE
她的人生啊 | /posts/her-life/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#597] [Announcements]
https://yuanfan.rbind.io/posts/her-life/

**Comment by yihui** [comment]
最近新听到一首蔡琴的《油麻菜籽》，这首歌跟我的年龄一样大。听了几遍之后，它就成了目前为止我最喜欢的蔡琴的歌，实在是太感人了。我想到了这篇日志，可以作为配乐。

**Reply by earfanfan** [reply]
我猜你会比较中意歌词，因为曲调跟你以往喜欢听的伤感情歌很像，相对而言，未必多么特别。不过我听了两遍都很难喜欢这首歌，歌词听了只觉得无比扎心，难受之极，大概是因为上周末我回去了一趟，又耐着性子听我妈絮叨了好几遍她的思想，又被她刺激成那种极端的“宁愿放弃生命也不愿意放弃自我意识”的心态，约莫再过几天等我慢慢把这波糟糕情绪处理好了，就能正常地听这首歌了吧。

**Reply by earfanfan** [reply]
啊，我处理好了，俺真棒，俺晚上回去再重新听听这首歌。

**Reply by yihui** [reply]
我太明白这种心态了，我也是如此倔强。

不知你看过《被讨厌的勇气》没有，如果没看过的话，不需要看全书，只看第三章的最后一节《人际关系“王牌”，握在你自己手里》就够了，看看是否有用。我忘了我之前是在哪里看到的“抑郁是因为以为自己没有选择”这句话（应该不是在这本书中），但这本书就是告诉你，你手里有选择权。

回到《油麻菜籽》这首歌，本来刚开始我并不知道歌词里说的“你”“我”分别指谁，我看到它是同名电影的主题曲之后，简单搜了一下电影简介，明白了李宗盛指的是电影中的母女（“你”是母亲、“我”是女儿）。我之所以想到你这篇日志，是因为倒数第二段简直就是油麻菜籽的精准刻画，而你自己有点像电影中那个反抗的菜籽二代。如果生活会按电影演的那样的话，最终也许你也会反抗成功。我自己一般不看电影，你若有兴趣可以找来看看，再安利给你妈，看她有什么感想。

另外，我觉得闽南语里的一些方言和比喻既质朴又形象，就比如这个油麻菜籽，还有袁哲生写过好多遍的“一枝草一点露”，都带着一股子底层劳动人民的命运味道。

**Reply by earfanfan** [reply]
说来真巧，我这两天正好开始看《被讨厌的勇气》，里面哲人与青年有五夜的对话，我才看完了第一夜，后面慢慢会看到你说的那节的。

其实我早就从原来的家庭中独立出来了，并无所谓反抗。我也不会安利我妈看这部电影，从我现在的视角看，她经过无数次的自我洗脑才达到今天这种像牛马一样苦累大半生还觉得幸福快乐的状态，我不想剥夺她的幸福。你可能不太了解我妈这样快六十岁的农村妇女的心态，说不定还对她们抱有撒奇奇怪怪的刻板印象（ps此处有顾虑怪尝试冒泡出来解释一大通，我给摁回去了），以我的观察，她那种受苦受累的生活里最需要的可能只是一种对她无私付出的赞美，倘若我鼓励她去寻找自我意识，我不知道精神世界相对贫瘠的她能不能承受得住那种三观崩溃又重塑的打击。我只决定从一些小细节上鼓励她尝试接触新事物，借用科技进步解放双手，比如她家明明有洗衣机还坚持手洗，我给她多讲讲要用洗衣机。

说到方言，我前些天去看了个歌舞戏剧类比赛，有些触动好想“打方叉”，然而我搜了遍都不知道这个音对应的字是撒。说与你知，看看你晓不晓得是撒。大概就是，一个人先站直，然后把双臂向上伸直，然后就类似达芬奇名画《维特鲁威人》那样，在地上滚一圈，但这个滚不是背部着地，而是双手撑一下，身体翻转一圈，然后又站起来。我猜肯定有个专业术语，只是我不知道。

**Reply by yihui** [reply]
你的肉身确实是独立出来了，但精神没法完全独立出来；耳朵关不上，唠叨还是能灌进去。电影里的母亲是反对女儿的婚恋，你那里是催促，一样都是对下一代意志的干涉。

我的意思不是说你去尝试改变她做牛做马的意识，那是改不了的，而且如你所说，要是真改了，麻烦会更大（需要整个大家庭的意识都相应更新，否则只能是鸡犬不宁）。我的意思是她有没有可能意识到她自己做牛做马是一回事，而你要不要做牛做马完全是另一回事，你们的选择应该是独立的，两种选择都可以幸福。当然，我知道这肯定也是很难的，你已经这样撞墙无数遍了。

“打方叉”我们方言里有，不过我们叫“打翻叉”（“叉”大概是指腿朝天的时候是叉形的）。我小时候一直没学会，很羡慕会打翻叉的同学，但只有女生会打。学名叫[侧手翻](https://zh.wikihow.com/%E5%81%9A%E4%BE%A7%E6%89%8B%E7%BF%BB)。

**Reply by earfanfan** [reply]
按照才看的《被讨厌的勇气》，从弗洛伊德原因论转向阿德勒的目的论来分析的话，俺这个半吊子也可以选择不把母女之间的思想观念差异看做一个大难题，而是可以去想想冲突发生时各自不肯妥协的目的，以及解决办法。

首先呢，有一点是很好的，那就是母女之间是可以正常沟通的，我跟我妈说她提催婚这件事其实是她自己的心态问题，是她自己要面对的考验，而不应该把她的焦虑全放到我身上来让我也跟着抑郁难受，所以她要减少提此事的频率才是当母亲该做好的，哈哈，我妈确实是有在克制的，她在这一点上是有进步的，值得肯定。而我呢，每次我妈稍微一提就情绪波动，现在也在练习先把情绪收好，认真听完她讲的，然后再私下悄悄处理糟糕情绪。总之就是避免正面冲突，不然的话很容易会变成互相伤害。

其次呢，她催促的目的并不是因为觉得我年纪大了没结婚会让她感到丢脸之类的，明面上说的是我没结婚她的人生任务没完成，实际底层仍然是对我的一种担忧以及愧疚。按照我妈那辈的传统观念，家庭传承是“单系传承”，父母的一生都应该奉献给儿子，而女儿会加入别人家的家庭而寻得新的依靠，现状是我爸妈把一生所有奉献给儿子，可是女儿并没有结婚寻找到一个完整的家庭可依靠，所以我妈会担忧和愧疚。而我每次愤怒的目的，其实跟孩子做错小事父母愤怒发脾气是一个道理，那就是试图用“愤怒”这个方式来胁迫对方，快速达到使对方妥协的目的，而忽视耐心沟通。

最后呢，我妈其实是无意识地催促，我感觉到她真得认识不到我俩之间的思想差异，许多时候她其实只是在表达对我的担忧，以及自己无法补偿我的那种分身乏术。上周我准备离开她家在收拾东西的时候，她忽然从坐着看电视开始起身去洗东西去了，我感觉她这种突然的开始忙碌实际上很可能是不愿意面对我的离去。仔细想一想，我受过很好的教育，看过很多心理学哲学的书籍，但是直至今日才开始试着慢慢地理解她的言行，而她一生忙碌一直都活在一个很狭窄的环境里，也没撒机会接触广阔天地，其实理应是我来好好地理解和支持她的，毕竟我的生命来源于她，能安稳地读书也是她辛苦打工换来的。所以，我下一步该练习的，也许应该是只把那些她无意识的催促之语当做是短暂的不愉快，不要总搁在心里抑郁来抑郁去的。也许会有一种新的可能，我可以试着帮助她去克服她的焦虑，试着让她以一种新的方式通过独属于她的考验。当然，我仍然不打算把我的思想灌输给她，只是也许她也可以意识到女儿没有结婚也并不用那么担忧，她的人生任务其实没有全部完成也不用遗憾。

**Reply by earfanfan** [reply]
另，为撒你的记忆里是只有女生会打呢，我的记忆里好像是男孩女孩都会，而且可以比较一下谁打的多，哈哈哈，难道这就是80后和90后的差异之一么？

话说我小时候，打翻叉通常是用在玩跳高这个游戏里，大致就是一群小屁孩，先“搓办撮办搓”（ps我也不知道这咋描述，约莫是围在一起伸出手心手背）分两帮，我们帮和他们帮。最开始他们帮派两个小孩牵那根皮筋，皮筋高度最开始是膝盖，我们帮全部跳过去了就升级，最后可以一直升到头顶上去。最后几关太高，一般都是只能打翻叉才能过去……也有特殊的，因为一群小屁孩不分年龄，有很小的小孩，就是一个先打翻叉翻过去，用卷起来的裤腿卡住皮筋，这样高度变矮了，长得矮的就可以跳过去了。

**Reply by yihui** [reply]
我看到打翻叉时第一个回忆也是跳皮筋，皮筋拉倒脖子那么高的时候，女生可以打翻叉把皮筋压下来，但我们班的男生好像都没这个身手，不知道为什么，可能是莫名其妙形成了“打翻叉是女孩子家家做的事情”这样一种刻板印象。那时候跳皮筋真是高低宽窄都可以推到极限状态，高可以上头，低可以挂到鞋帮子、离地大概就两厘米的样子，跳皮筋的人最后要拼命让脚从皮筋底下穿过去。我记得步骤是这样的：

1. 跳踩第一根（不能碰第二根）；
2. 跳踩第二根（不能碰第一根）；
3. 同时跳踩两根，退踩第一根，下来；
4. 双脚从第一根下面拱过去，并跨过第二根；
5. 跳转身压住第一根；
6. 往前迅速拱到第二根下面，再挑出去。

如果步骤 5 不小心同时压住了两根皮筋，那么就小心地把第二根皮筋从鞋底松出去。我对这个印象最深刻，因为有时候两根皮筋太近了太难分开，一不小心就把两根皮筋都松掉了。

很高、很低、很宽、很窄的皮筋都难跳。还有单脚跳的版本（用一只脚完成前面的那些动作）。放学后或假期里我就经常拿旧秋裤上拆下来的皮筋绷在两把椅子的腿上在家刻苦练习，这种活动让人迷之上瘾。

说起打翻叉，我上初一的时候学会了靠墙倒立，还一度想练习倒立走路，无奈没有练出来。记得有一次在操场上，我鼓起勇气凭空倒立，结果翻了过去，来了个死鱼打挺、摔得好疼，打那以后就不敢这样练了。还记得隔壁寝室有个男生见我能手撑地、头朝下猛地把身子翻过来靠墙倒立，他不会这样翻，结果这位兄台就撑着地，先把头顶在地上，然后凭胳膊的力量把自己的身体撑起来、让头离地，我也是由衷佩服他的臂力。体育课上老师还教了我们一种三角倒立法，就是双手加上头作为三角支点，然后渐渐把身子倒着伸直。我学会后，没事就在教室里的凳子上这样倒立。到现在我还会这两种倒立，而打翻叉我似乎是前年在家试图练过几天，依然有点心虚怕摔，大概能打个龙虾式的翻叉（身体不敢完全舒展伸直）。

**Reply by earfanfan** [reply]
我小时候打的全都是你说的这种龙虾翻叉……前边我说的跳高是牵一根皮筋跳过去，不是跳皮筋。但你描述的这一连串步骤让我想起来了类似的，本来应该是有个名字的，但我记不得，哪天想起来再说与你知。我们那跳这个的步骤是酱紫的，为方便描述，你可想象你站在皮筋前，那个皮筋是被两边撑开呈长方形，面前就是长方形的两条长边，离你的人近的那根称为A线，远的那根称为B线，开始跳：

1. 跳到A线上，双脚踩住A线，一样不能碰到B线。
2. 继续从A线跳到B线上，双脚踩住B线，同样不能碰到A线。
3. 从B线继续往前跳下去，离开B线，再跳一次转身，从线都在身后转成线都在面前，当然是不能碰到线，身体也不能多动的，此时B线离人身更近。
4. 接着站在B线外，一口气凌空跳过两根线，落地的位置与最开始第一步之前一样，依旧是面前两根线，A线更近。
5. 再接着从A线下的位置跳到B线上，双脚踩在B线上，而A线在脚的上方。
6. 这一步通常翻车比较多，那就是再转身跳一次，变成双脚离开B线踩在A线上，同时B线还被A线压在下面。
7. 这一步翻车也多，主要看A线被双脚踩成两段与B线形成的那个三角形是偏大还是偏小，因为要再跳一次双脚再离开A线踩到B线上，要是那个三角形偏大，那么B线很远就不好跳上去了。
8. 最后一步，转身凌空跳过A线，回到初始状态。

我说的这个就是纯跳，通常是在皮筋高度偏低的时候玩，大概就是在脚踝、小腿中部、膝盖的高度可以玩的，双脚是一起行动的，还有别的跳法是双脚需要分开行动的。你说的这种同时用一只脚踩住两根线，再用脚“腻（第一声）啊腻”（这里也不知道普通话咋形容），大概就是用脚碾压脚底的两根线，放掉一根，留下的这根依然会被踩成两段和放掉的那根形成三角形，这种玩法我水平也不咋滴。

**Reply by earfanfan** [reply]
话说我又想起另一种小时候玩的游戏，未曾记到我的角落里。这个游戏的名字念做“潜掰”，音“潜”是指一个小孩折起一只脚，只用另一脚单脚跳，音“掰”是指一种物体，通常是偏扁一点还有点厚度的石头，踢一脚给它一个力，它能往前飞却又不会飞太远……哈哈，“潜掰”组合起来就是，小孩子单脚挑起来去踢一个物体，通过使出去的力道不同控制此物体前进的远近距离，最终让这个物体飞到小孩子想让它飞到的格子里面去。这个游戏也是男孩女孩都可以一起玩的，只需要在一个平整一点的土地上，用石头或棍子在地上画出格子，分完帮就可以开始玩了。那个格子的画法我记不大清了，大致是一个大大的长方形被三横一竖分成八个均等的格子，好像还是要计分的，貌似还有一个格子外面会有一个半圆，要是一不小心把“掰”踢进去了，那么之前累积的分数就全清零了。

**Reply by earfanfan** [reply]
话又说胡来，前天我的好友姚想要为她的公众号起个新名字，我才突然反应过来我的博客也是没名字的啊，而我的博客没名字最初的锅是因为看你和列弛的博客就是没名字的，只有博主的名字，明明也看过落园和怡然轩，但是从没想到过要给自己的博客起个名字。
